<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-06T13:53:13Z" name="Default suite" finished-at="2022-12-06T13:53:37Z" duration-ms="23422">
    <groups>
    </groups>
    <test started-at="2022-12-06T13:53:13Z" name="Default test" finished-at="2022-12-06T13:53:37Z" duration-ms="23422">
      <class name="TestNG.Khan_TestNG">
        <test-method is-config="true" signature="beforeclass()[pri:0, instance:TestNG.Khan_TestNG@54422e18]" started-at="2022-12-06T13:53:13Z" name="beforeclass" finished-at="2022-12-06T13:53:20Z" duration-ms="6705" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeclass -->
        <test-method is-config="true" signature="beforemethod()[pri:0, instance:TestNG.Khan_TestNG@54422e18]" started-at="2022-12-06T13:53:20Z" name="beforemethod" finished-at="2022-12-06T13:53:21Z" duration-ms="686" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method signature="test1()[pri:0, instance:TestNG.Khan_TestNG@54422e18]" started-at="2022-12-06T13:53:21Z" name="test1" finished-at="2022-12-06T13:53:25Z" duration-ms="4563" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method is-config="true" signature="aftermethod(org.testng.ITestResult)[pri:0, instance:TestNG.Khan_TestNG@54422e18]" started-at="2022-12-06T13:53:25Z" name="aftermethod" finished-at="2022-12-06T13:53:26Z" duration-ms="793" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test1 status=SUCCESS method=Khan_TestNG.test1()[pri:0, instance:TestNG.Khan_TestNG@54422e18] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
        <test-method is-config="true" signature="beforemethod()[pri:0, instance:TestNG.Khan_TestNG@54422e18]" started-at="2022-12-06T13:53:26Z" name="beforemethod" finished-at="2022-12-06T13:53:30Z" duration-ms="4036" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method signature="test2()[pri:0, instance:TestNG.Khan_TestNG@54422e18]" started-at="2022-12-06T13:53:30Z" name="test2" finished-at="2022-12-06T13:53:35Z" duration-ms="4972" status="FAIL">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)
at TestNG.Khan_TestNG.test2(Khan_TestNG.java:70)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method is-config="true" signature="aftermethod(org.testng.ITestResult)[pri:0, instance:TestNG.Khan_TestNG@54422e18]" started-at="2022-12-06T13:53:35Z" name="aftermethod" finished-at="2022-12-06T13:53:36Z" duration-ms="1482" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test2 status=FAILURE method=Khan_TestNG.test2()[pri:0, instance:TestNG.Khan_TestNG@54422e18] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
        <test-method is-config="true" signature="afterclass()[pri:0, instance:TestNG.Khan_TestNG@54422e18]" started-at="2022-12-06T13:53:36Z" name="afterclass" finished-at="2022-12-06T13:53:37Z" duration-ms="147" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterclass -->
      </class> <!-- TestNG.Khan_TestNG -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
